charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# Architecture++
```

Comment:
Perhaps `s/Architecture++/DCP Architecture/` - maybe even _DCP Architecture Team_, the playful **++** notation in the presentation was intended to convey the next small increment of an existing group. Of course, if the technical leads enjoy **++**, then by all means keep it.